plugins/woocommerce/includes/admin/class-wc-admin-post-types.php

```diff
@@ -139,7 +139,7 @@ public function post_updated_messages( $messages ) {
 			9  => sprintf(
 				/* translators: 1: date 2: product url */
 				__( 'Product scheduled for: %1$s. <a target="_blank" href="%2$s">Preview product</a>', 'woocommerce' ),
-				'<strong>' . date_i18n( __( 'M j, Y @ G:i', 'woocommerce' ), strtotime( $post->post_date ) ) . '</strong>',
+				'<strong>' . date_i18n( __( 'M j, Y @ G:i', 'woocommerce' ), ($post->post_date ? strtotime( $post->post_date ) : false) ) . '</strong>',
```

Comment:
This change isn't necessary. In this context, `$post` is guaranteed to be a valid object (including post date).

Reply:
indeed, done